.rubocop.yml

```diff
@@ -5,7 +5,7 @@ AllCops:
 Layout/LineLength:
   Max: 125
   Exclude:
-    - "spec/**/*.rb"
+    - "spec/**/**/*.rb"
```

Comment:
the `**` should handle all recursive subdirectories? I don't think this change is required.